pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -289,14 +289,30 @@ const (
 	// at the same time. During a failover, it is crucial to ensure that the old
 	// application is removed before creating a new one to avoid duplicate
 	// processing and maintaining state consistency.
+	// Deprecated: in favor of PurgeModeDirectly.
```

Comment:
Please add more instructions to this deprecation, including why this is deprecated, if it is effective, when it will be removed.

Reply:
Thanks~, can you help review it again?